src/OpenLoco/ScenarioObjective.h

```diff
@@ -8,19 +8,27 @@ namespace OpenLoco::Scenario
 #pragma pack(push, 1)
     struct Objective
     {
-        Scenario::ObjectiveType type;        // 0x000418 (0x00526230)
-        uint8_t flags;                       // 0x000419 (0x00526231)
-        uint32_t companyValue;               // 0x00041A (0x00526232)
-        uint32_t monthlyVehicleProfit;       // 0x00041E (0x00526236)
-        uint8_t performanceIndex;            // 0x000422 (0x0052623A)
-        uint8_t deliveredCargoType;          // 0x000423 (0x0052623B)
-        uint32_t deliveredCargoAmount;       // 0x000424 (0x0052623C)
-        uint8_t timeLimitYears;              // 0x000428 (0x00526240)
+        Scenario::ObjectiveType type;  // 0x000418 (0x00526230)
+        uint8_t flags;                 // 0x000419 (0x00526231)
+        uint32_t companyValue;         // 0x00041A (0x00526232)
+        uint32_t monthlyVehicleProfit; // 0x00041E (0x00526236)
+        uint8_t performanceIndex;      // 0x000422 (0x0052623A)
+        uint8_t deliveredCargoType;    // 0x000423 (0x0052623B)
+        uint32_t deliveredCargoAmount; // 0x000424 (0x0052623C)
+        uint8_t timeLimitYears;        // 0x000428 (0x00526240)
+    };
+#pragma pack(pop)
+    static_assert(sizeof(Objective) == 0x11);
+    Objective& getObjective();
+
+#pragma pack(push, 1)
+    struct Objective2
```

Comment:
ObjectiveProgress?

Reply:
sounds good, I didn't know what to call it!

Reply:
done